src/disk_interface.cc

```diff
@@ -60,11 +60,11 @@ int MakeDir(const string& path) {
 TimeStamp TimeStampFromFileTime(const FILETIME& filetime) {
   // FILETIME is in 100-nanosecond increments since the Windows epoch.
   // We don't much care about epoch correctness but we do want the
-  // resulting value to fit in an integer.
+  // resulting value to fit in a 64-bit integer.
```

Comment:
According to the first link, you shouldn't treat `FILETIME` as a 64-bit integer directly because of possible alignment issues.